products/kms/terraform.yaml

```diff
@@ -100,6 +100,37 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       encoder: templates/terraform/encoders/kms_crypto_key.go.erb
       update_encoder: templates/terraform/update_encoder/kms_crypto_key.go.erb
       extra_schema_entry: templates/terraform/extra_schema_entry/kms_self_link.erb
+  SecretCiphertext: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+
+      ~> **NOTE**: Using this resource will allow you to conceal secret data within your
+      resource definitions, but it does not take care of protecting that data in the
+      logging output, plan output, or state output.  Please take care to secure your secret
+      data outside of resource definitions.
+    id_format: "{{crypto_key}}/{{ciphertext}}"
+    supports_indirect_user_project_override: true
+    exclude_import: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "kms_secret_ciphertext_basic"
+        primary_resource_id: "my_password"
+        vars:
+          instance_name: "my-instance"
+        skip_test: true
+    properties:
+      cryptoKey: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
```

Comment:
i think url_param should cover this, but am also fine leaving this in since it's easier to see what is being ignored read from the terraform file

Reply:
Added a comment anyway just to be extra clear